dev/com.ibm.ws.jdbc.metatype/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -145,7 +145,7 @@ valTimeout.desc=Specifies a timeout for validation of pooled connections. When s
 # Vendor properties
 
 properties=Generic JDBC Driver Properties
-properties.desc=List of JDBC vendor properties for the data source. For example, databaseName="dbname" serverName="localhost" portNumber="50000".
+properties.desc=List of JDBC vendor properties for the data source. For example, databaseName="dbname" serverName="localhost" portNumber="50000". This generic properties list can be used when no vendor-specific properties list type is available for the JDBC driver that you are using. Do not specify multiple properties lists on a data source.
```

Comment:
I would suggest changing the following:
```
Do not specify multiple properties lists on a data source.
```
to
```
Do not specify multiple properties lists on a data source.  Instead, place all property values inside a single properties list.
```

Reply:
Thanks for the suggestion.  It is updated under https://github.com/OpenLiberty/open-liberty/pull/20988/commits/e004f1c660be2d2faa14a5da2ff55a3faa671949